### DISCUSSION TITLE
Missing poll.cancel() ?

### DISCUSSION THREAD

**Discussion by TutorExilius** [TwitchIO/TwitchIO#410] [Questions]
Hello,

I am not sure, but could that be that a `self._poll.cancel()` is missing here?

https://github.com/PythonistaGuild/TwitchIO/blob/9ae92512990bb23ee5c8e5d716e9a52a55953b5e/twitchio/ext/pubsub/websocket.py#L168C2-L168C3

**Comment by IAmTomahawkx** [comment]
Hey @TutorExilius , not sure how this got missed, but no there isn't. Cancelling the poll task there would risk cancelling the reconnect operation being performed from within the poll task, so we simply let it run its course (once it's hit that particular branch of code it'll die anyways after reconnecting)
